pkg/client/errors.go

```diff
@@ -21,3 +21,12 @@ func PrintErrorAndExit(format string, args ...interface{}) {
 	fmt.Fprintln(os.Stderr, color.RedString(format, args...))
 	os.Exit(1)
 }
+
+func PrintConnectionErrorAndExit(err error) {
+	PrintErrorAndExit(
+		"A connection error occour before you command run.\n" +
+		"Please, try again.\n\n" +
+		"Error details: " +
+		GetErrorMsg(err),
+	)
+}
```

Comment:
I guess this is better :  `A connection error occurred. Please...`
Mixed feelings...for me the error details will only create confusion. @drgarcia1986 What do you think?

Reply:
I agree ☝️

Reply:
You both think it's better without error details?

Reply:
So, should return the cluster config or just the new message (`A connection error occurred. Please...`)?

Reply:
current cluster config is good to know "where am I?"